editor-desktop/src/com/talosvfx/talos/TALOS_BUILD.java

```diff
@@ -0,0 +1,67 @@
+package com.talosvfx.talos;
```

Comment:
Should be in editor project, not editor-desktop

Reply:
oh okay, lemme fix

Reply:
shall I also move the task "generateBuildConfig" from :editor-desktop to :editor @Tom-Ski

Reply:
yes

Reply:
fixed